js/src/block-editor/controls/text.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
```

Comment:
Hey! How is this useful? Noticed you included it in all the controls.

Reply:
Yeah, it's a little dirty having this in every React file.

But it lets VS Code do type checking. This has revealed some incorrect props, like [here](https://github.com/studiopress/genesis-custom-blocks/pull/63/files#diff-3123e4746cd8f36584d725dcc80b1ef6041d37a82735bc5ef6a1b71bbb50899aL11)